types/config.d.ts

```diff
@@ -107,6 +108,14 @@ declare global {
       }
       export interface Group extends GroupJson {}
 
+      export interface Plugin {
+        /** Optionally provide more audits to run in addition to those specified by the base config. */
+        audits?: Array<{path: string}>;
```

Comment:
Can we guess as to what properties we'd add to `audits[]` in the future? Or is it meant to exactly match `Config.AuditJson` so merging is straightforward?

I suggest that allowing strings (`Array<string|{path:string}>`) would be ergonomic, where the string value is aliased to `{path: <string>}`

Reply:
I see that `Config.AuditJson` itself allows for `string`.

Reply:
> Or is it meant to exactly match `Config.AuditJson` so merging is straightforward?

exactly

> `Array<string|{path:string}>`

yeah, `Config.AuditJson` was originally just an array of strings (that's still the form used in the default configs), but we added the object form when we added audit options. I wanted to support just a singular form for the initial implementation PR to keep things focused and readable, and we'll likely need to support audit options in plugins, so I picked the object form to start with.